krator/src/runtime.rs

```diff
@@ -377,19 +377,30 @@ async fn run_object_task<O: Operator>(
     let state: O::InitialState = Default::default();
     let (namespace, name) = {
         let m = manifest.latest();
+        let name = match &m.meta().name {
```

Comment:
Here we do definitely need the name, so I've gone ahead and implemented a real check.